lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm

```diff
@@ -1142,7 +1150,7 @@ sub _generate_standard_files {
             $self->$lq_accessor($lq_result);
 
             my $f = $lq_result->_file_for_type($variant_type);
-            Genome::Sys->create_symlink($lq_result->path($f), $self->output_directory . "/$f");
+            Genome::Sys->create_symlink($lq_result->path($f), $self->_temp_staging_directory . "/$f");
```

Comment:
I have converted this and some additional string concatenation to use `File::Spec`.